trie/transition.go

```diff
@@ -63,7 +66,23 @@ func (t *TransitionTrie) TryGet(addr, key []byte) ([]byte, error) {
 		return val, nil
 	}
 	// TODO also insert value into overlay
-	return t.base.TryGet(nil, key)
+	rlpval, err := t.base.TryGet(nil, key)
+	if err != nil {
+		return nil, err
+	}
+	if len(rlpval) == 0 {
+		return nil, nil
+	}
+	// the value will come as RLP, decode it so that the
+	// interface is consistent.
+	_, content, _, err := rlp.Split(rlpval)
```

Comment:
in the MPT, the data is serialized using RLP. The verkle tree does not do this, so to keep it consistent, deserialize here.